src/csv_overrides.py

```diff
@@ -14,7 +14,11 @@
 )
 
 
-def init_csv_and_watch_changes(filename: str, default_values: dict[str, dict]):
+def init_csv_and_watch_changes(
+    filename: str,
+    default_values: dict[str, dict],
+    extra_acceptable_values: list[str] = None,
```

Comment:
the `extra_acceptable_values` makes it so we don't complain if they have customized the name of a style that they're not currently using